.github/workflows/tapioca-update.yml

```
@@ -0,0 +1,57 @@
+name: Tapioca Update
+on:
+  push:
+    paths:
+      - .github/workflows/tapioca-update.yml
+  schedule:
+    - cron: '0 0 */3 * *' # every three days
+env:
+  HOMEBREW_SORBET: 1
+jobs:
+  update-tapioca:
+    if: github.repository == 'Homebrew/brew'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Configure Git user
+        uses: Homebrew/actions/git-user-config@master
+        with:
+          username: BrewTestBot
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6.3
+
+      - name: Update Tapioca Definitions
+        run: |
+          cd "$GITHUB_WORKSPACE/Library/Homebrew"
+          bundle install
+          bundle exec tapioca sync --exclude json
+          bundle exec srb rbi hidden-definitions
+          if ! git diff --exit-code -- sorbet; then
```

```suggestion
          cd "$GITHUB_WORKSPACE/Library/Homebrew"
          bundle install
          bundle exec tapioca sync --exclude json
          bundle exec srb rbi hidden-definitions
          if ! git diff --exit-code -- sorbet; then
```

I presume this can all be done by https://github.com/Homebrew/brew/pull/8289? If not: it'd be good to add so a single command can be run.

---

> It is not. If someone can let me know what rights it needs: I can add it.

We need this Action to be able to use a `BrewTestBot` PAT to push commits to a branch of this repo and raise PRs. That is, `public_repo`.

---

@vidusheeamoli How would you feel about integrating these Tapioca update commands into `brew typecheck`?

---

@issyl0  On it!